modules/dreamview/frontend/src/renderer/obstacles.js

```diff
@@ -362,24 +432,36 @@ export default class PerceptionObstacles {
       new THREE.Vector3(0.5, 0, 1),
       new THREE.Vector3(-0.5, 0, 1),
     ];
-    const extrusionFace = solid
-      ? drawSegmentsFromPoints(points, DEFAULT_COLOR, LINE_THICKNESS)
-      : drawDashedLineFromPoints(points, DEFAULT_COLOR, LINE_THICKNESS, 0.1, 0.1);
+    let extrusionFace = null;
+    if (_.isEqual(type,'SOLID_FACE')) {
```

Comment:
switch (type) {
   case FACE_TYEP.SOLID_FAC:
      ....
      break;
   case xxxx
   ....
}

same for the rest